### PR TITLE
Recommended recipes page: recipe image and icons

### PR DESCRIPTION
**Pull request by azhara-a**
- **Mobile:**
Screenshot 2020-09-11 at 12 52 35 - Edited
- **Desktop:**
Screenshot 2020-09-11 at 4 35 30 PM

### CODE REVIEW

**Review by mam249 — COMMENTED** [review]
LGTM

File: backend/src/main/java/com/google/sps/data/Recipe.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
   private final String time;
   private final String calories;
   private final String difficulty;
+  private final String image;
```
Comment (lminchillo) [inline backend/src/main/java/com/google/sps/data/Recipe.java:26]: nit: I think `image` is a bit of a weird name -- this String is actually the URL to the image, right? If that's the case I'd suggest something more descriptive like `imageUrl`

**Review by lminchillo — APPROVED** [review]
LGTM

**Review by mugdhalakhani — APPROVED** [review]
LGTM, thanks!

**Pull request merged**